docs/Homebrew-linuxbrew-core-Maintainer-Guide.md

```diff
@@ -73,8 +73,8 @@ changes that the upstream Homebrew developers have made.
 brew merge-homebrew --core
 ```
 
-Merging all the changes from upstream in one go is usually
-undesirable since our build servers will time out. Instead, attempt
+Merging all the changes from upstream in one go is risky since it's
+harder to keep track of all the active builds. Instead, attempt
```

Comment:
I think "risky" doesn't match "harder" -- I'd say something is risky when there is a higher chance of a failure.

Reply:
Yes, just go for "harder" :)